.travis.yml

```diff
@@ -6,17 +6,16 @@ rust:
 - stable
 matrix:
   include:
-  - rust: "1.26.2"
-    before_script:
-    - rustup component add rustfmt-preview --toolchain 1.26.2
+  - rust: nightly-2018-07-22
+    env: CLIPPY=YESPLEASE
     script:
-    - rustfmt -V
-    - cargo fmt -- --write-mode diff
-  - rust: nightly-2018-06-19
-    before_script:
-    - cargo install clippy --vers 0.0.209
+    - rustup component add clippy-preview
+    - cargo clippy
```

Comment:
Do you want the previous behavior with `-D warnings`, or is it fine to just print warnings?

Reply:
Oh, good catch!